addon/mixins/spread.js

```diff
@@ -108,9 +108,11 @@ export default Mixin.create({
    * @param {object} spreadableHash - the hash to spread
    */
   _defineSpreadProperties (spreadProperty, spreadableHash) {
+    const supportedSpreadableTypes = A(['instance', 'object'])
+
     assert(
       `${spreadProperty} requires an Ember object or primitive object`,
-      ['instance', 'object'].includes(typeOf(spreadableHash))
+      supportedSpreadableTypes.includes(typeOf(spreadableHash))
```

Comment:
Browsers that actually render DOM to a screen to be seen by a human :)

I'm all for doing what's reasonable and nice, and in this particular case, it's not a very big impact. However, I've had a lot of very negative experiences with having to code things without using language features I'd like to, just to support a test environment that is lagging behind my production/dev environment. I was quite glad to stop using phantomjs as soon as support for running automated tests in real browsers became prevalent. If we agree to support phantomjs here, we're agreeing to support phantomjs in all our projects, and that can introduce a lot of overhead for us in our daily operations. 

Speaking of reasonable and nice, I think avoiding the use of profanity in pull requests falls into that category.